.github/workflows/publish.yml

```
@@ -22,6 +22,9 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install -y libssl-dev
+          wget https://luarocks.org/releases/luarocks-3.3.1.tar.gz
+          tar zxpf luarocks-3.3.1.tar.gz
+          cd luarocks-3.3.1 && ./configure --with-lua-include=/usr/include/lua5.1 && make && sudo make install
```

```suggestion
          sudo apt-get install -y libssl-dev luarocks
```
isso não funcionaria?